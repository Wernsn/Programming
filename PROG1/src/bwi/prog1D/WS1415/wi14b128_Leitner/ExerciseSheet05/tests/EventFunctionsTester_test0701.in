7
title
2000
1
1
1
0
7
title
2000
1
1
5
1
0

