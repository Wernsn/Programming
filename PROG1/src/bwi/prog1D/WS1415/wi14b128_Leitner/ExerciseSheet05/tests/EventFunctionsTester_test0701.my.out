display: invalid event

enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: Saturday, January 1st 2000, title, 1

invalid event

display: invalid event

enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: title, 5

invalid event

